test/support/rails_app.rb

```diff
@@ -6,6 +6,8 @@ module ActiveModelSerializers
       config.active_support.test_order = :random
       config.action_controller.perform_caching = true
       config.action_controller.cache_store = :memory_store
+
+      config.filter_parameters += [:password]
```

Comment:
do the tests fail with just this change, and not the fix in `lib/active_model_serializers/test/schema.rb` ?